locksmith/src/utils/keyData.ts

```diff
@@ -48,7 +48,7 @@ export default class KeyData {
         expiration: keyExpiration ? parseInt(keyExpiration) : undefined,
         tokenId: key?.tokenId,
         owner: key?.owner,
-        minted: key?.createdAt ? parseInt(key.createdAt) : undefined,
```

Comment:
I think `createdAtBlock` is the block number... not the timestamp.
The point of this issue is to add it as metadadata so that it is visible on OpenSea per these specs: https://docs.opensea.io/docs/metadata-standards